source/content.css

```diff
@@ -577,6 +577,10 @@ div.inline-comment-form .form-actions,
 	width: 420px;
 }
 
+.subnav-links .subnav-item {
```

Comment:
This needs a comment on why it's there and what it does.

Reply:
I think it should also be moved to a more relevant place in the file: https://github.com/tanmayrajani/refined-github/blob/27fc4a338a8e27f7e84cbd2de1b7cb4499569755/source/content.css#L575